test/simple-test/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.9)
+project(tests)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/../cmake" ${CMAKE_MODULE_PATH})
+
+find_package(MPI REQUIRED)
```

Comment:
I meant:
```
find_package(ParMETIS REQUIRED)
```

Reply:
Oh ok; I see; the `puml.cpp` doesn't use ParMETIS—I just forgot to remove the header/linking references. They're gone now.